package.json

```diff
@@ -175,7 +175,7 @@
     "@maticnetwork/maticjs-ethers": "^1.0.2",
     "axios": "^1.2.1",
     "buffer": "^6.0.3",
-    "ethers": "^5.7.2",
+    "ethers": "5.7.2",
```

Comment:
note: we don't want this version to increase per https://docs.ethers.org/v5/api/signer/#Signer-signTypedData. think it's okay to fix here since 5.7.2 is the last release of v5